src/main.rs

```diff
@@ -189,7 +189,7 @@ fn main() {
                         }
                     }
                     cfg.language_server
-                        .insert(language.command.clone(), language);
+                        .insert(format!("{}:{}", language_id, language.command), language);
```

Comment:
Great find, thanks!

I wonder at what point we should start rejecting old configs.
Our plan is to deprecate the current TOML config as well in favor of using native Kakoune options and hooks.

I'm working on fixing the unrelated CI failures.